Gemfile

```diff
@@ -4,12 +4,13 @@ gemspec
 
 group :development, :test do
   gem 'between_meals', :git => 'https://github.com/facebook/between-meals'
-  gem 'chef-dk', '~> 3.13.0'
-  gem 'chef-zero'
+  gem 'chef', '= 16.18.0'
+  gem 'chef-zero', '>= 15.0.11'
   gem 'openssl'
   gem 'rspec-core', '>= 3.0.0'
   gem 'rspec-expectations', '>= 3.0.0'
   gem 'rspec-mocks', '>= 3.0.0'
-  gem 'rubocop', '= 1.3.1'
+  gem 'rubocop', '= 1.25.1'
   gem 'simplecov'
+  gem 'webrick' # chef-zero dep
```

Comment:
we shouldn't explicitly set dependencies of our dependencies, `chef-zero` should have this in it's deps. We should only list our direct deps.